ToneMapper/TMGLViewController.m

```diff
@@ -68,16 +50,24 @@ - (void)viewDidAppear:(BOOL)animated {
   if (!self.glkView) {
```

Comment:
Done. 

Though this is quite bizarre, the obvious choice was using `viewDidLoad:` for this kind of behaviour, putting it in `viewDidAppear:` was a necessity since initialising them required the containing view's `bound` property that isn't correctly initialised in `viewDidAppear:`.

Due to your remark I tried moving it to `viewDidLoad:` and it worked - but I'm not sure what's different from before that makes it suddenly work fine. If you can shed light on this matter I'd be glad.